### ISSUE TITLE
2024-12-20, Papers for large language model, llm

### ISSUE THREAD

**Issue by 9tailwolf** [Arxiv-Sender/Arxiv-Sender#26]
# Today's Arxiv Papers

(1) [OpenEMMA: Open-Source Multimodal Model for End-to-End Autonomous Driving](http://arxiv.org/abs/2412.15208v1), 2024-12-19

(2) [LongBench v2: Towards Deeper Understanding and Reasoning on Realistic Long-context Multitasks](http://arxiv.org/abs/2412.15204v1), 2024-12-19

(3) [MMLU-CF: A Contamination-free Multi-task Language Understanding Benchmark](http://arxiv.org/abs/2412.15194v1), 2024-12-19

(4) [Face the Facts! Evaluating RAG-based Fact-checking Pipelines in Realistic Settings](http://arxiv.org/abs/2412.15189v1), 2024-12-19

(5) [LlamaFusion: Adapting Pretrained Language Models for Multimodal Generation](http://arxiv.org/abs/2412.15188v1), 2024-12-19

(6) [Data for Mathematical Copilots: Better Ways of Presenting Proofs for Machine Learning](http://arxiv.org/abs/2412.15184v1), 2024-12-19

(7) [HPC-Coder-V2: Studying Code LLMs Across Low-Resource Parallel Languages](http://arxiv.org/abs/2412.15178v1), 2024-12-19

(8) [Critical-Questions-of-Thought: Steering LLM reasoning with Argumentative Querying](http://arxiv.org/abs/2412.15177v1), 2024-12-19

(9) [Rethinking Uncertainty Estimation in Natural Language Generation](http://arxiv.org/abs/2412.15176v1), 2024-12-19

(10) [Prompt-A-Video: Prompt Your Video Diffusion Model via Preference-Aligned LLM](http://arxiv.org/abs/2412.15156v1), 2024-12-19

(11) [Language Models as Continuous Self-Evolving Data Engineers](http://arxiv.org/abs/2412.15151v1), 2024-12-19

(12) [Adaptive Pruning for Large Language Models with Structural Importance Awareness](http://arxiv.org/abs/2412.15127v1), 2024-12-19

(13) [Outcome-Refining Process Supervision for Code Generation](http://arxiv.org/abs/2412.15118v1), 2024-12-19

(14) [Qwen2.5 Technical Report](http://arxiv.org/abs/2412.15115v1), 2024-12-19

(15) [Associative memory inspires improvements for in-context learning using a novel attention residual stream architecture](http://arxiv.org/abs/2412.15113v1), 2024-12-19

(16) [Review-Then-Refine: A Dynamic Framework for Multi-Hop Question Answering with Temporal Adaptability](http://arxiv.org/abs/2412.15101v1), 2024-12-19

(17) [Nano-ESG: Extracting Corporate Sustainability Information from News Articles](http://arxiv.org/abs/2412.15093v1), 2024-12-19

(18) [ScamChatBot: An End-to-End Analysis of Fake Account Recovery on Social Media via Chatbots](http://arxiv.org/abs/2412.15072v1), 2024-12-19

(19) [ConfliBERT: A Language Model for Political Conflict](http://arxiv.org/abs/2412.15060v1), 2024-12-19

(20) [LLMs Lost in Translation: M-ALERT uncovers Cross-Linguistic Safety Gaps](http://arxiv.org/abs/2412.15035v1), 2024-12-19

(21) [Large Language Models and Code Security: A Systematic Literature Review](http://arxiv.org/abs/2412.15004v1), 2024-12-19

(22) [HSEvo: Elevating Automatic Heuristic Design with Diversity-Driven Harmony Search and Genetic Algorithm Using LLMs](http://arxiv.org/abs/2412.14995v1), 2024-12-19

(23) [RoboCup@Home 2024 OPL Winner NimbRo: Anthropomorphic Service Robots using Foundation Models for Perception and Planning](http://arxiv.org/abs/2412.14989v1), 2024-12-19

(24) [Chain-of-MetaWriting: Linguistic and Textual Analysis of How Small Language Models Write Young Students Texts](http://arxiv.org/abs/2412.14986v1), 2024-12-19

(25) [AI and Cultural Context: An Empirical Investigation of Large Language Models' Performance on Chinese Social Work Professional Standards](http://arxiv.org/abs/2412.14971v1), 2024-12-19

(26) [Movie2Story: A framework for understanding videos and telling stories in the form of novel text](http://arxiv.org/abs/2412.14965v1), 2024-12-19

(27) [Knowledge Injection via Prompt Distillation](http://arxiv.org/abs/2412.14964v1), 2024-12-19

(28) [Understanding the Dark Side of LLMs' Intrinsic Self-Correction](http://arxiv.org/abs/2412.14959v1), 2024-12-19

(29) [RobustFT: Robust Supervised Fine-tuning for Large Language Models under Noisy Response](http://arxiv.org/abs/2412.14922v1), 2024-12-19

(30) [Dehallucinating Parallel Context Extension for Retrieval-Augmented Generation](http://arxiv.org/abs/2412.14905v1), 2024-12-19

(31) [Multimodal Hypothetical Summary for Retrieval-based Multi-image Question Answering](http://arxiv.org/abs/2412.14880v1), 2024-12-19

(32) [Graph-Convolutional Networks: Named Entity Recognition and Large Language Model Embedding in Document Clustering](http://arxiv.org/abs/2412.14867v1), 2024-12-19

(33) [Think&Cite: Improving Attributed Text Generation with Self-Guided Tree Search and Progress Reward Modeling](http://arxiv.org/abs/2412.14860v1), 2024-12-19

(34) [DS$^2$-ABSA: Dual-Stream Data Synthesis with Label Refinement for Few-Shot Aspect-Based Sentiment Analysis](http://arxiv.org/abs/2412.14849v1), 2024-12-19

(35) [Mapping and Influencing the Political Ideology of Large Language Models using Synthetic Personas](http://arxiv.org/abs/2412.14843v1), 2024-12-19

(36) [Helping LLMs Improve Code Generation Using Feedback from Testing and Static Analysis](http://arxiv.org/abs/2412.14841v1), 2024-12-19

(37) [DynamicKV: Task-Aware Adaptive KV Cache Compression for Long Context LLMs](http://arxiv.org/abs/2412.14838v1), 2024-12-19

(38) [ObjVariantEnsemble: Advancing Point Cloud LLM Evaluation in Challenging Scenes with Subtly Distinguished Objects](http://arxiv.org/abs/2412.14837v1), 2024-12-19

(39) [Progressive Multimodal Reasoning via Active Retrieval](http://arxiv.org/abs/2412.14835v1), 2024-12-19

(40) [Answer Set Networks: Casting Answer Set Programming into Deep Learning](http://arxiv.org/abs/2412.14814v1), 2024-12-19

(41) [ResoFilter: Rine-grained Synthetic Data Filtering for Large Language Models through Data-Parameter Resonance Analysis](http://arxiv.org/abs/2412.14809v1), 2024-12-19

(42) [Disentangling Reasoning Tokens and Boilerplate Tokens For Language Model Fine-tuning](http://arxiv.org/abs/2412.14780v1), 2024-12-19

(43) [ALKAFI-LLAMA3: Fine-Tuning LLMs for Precise Legal Understanding in Palestine](http://arxiv.org/abs/2412.14771v1), 2024-12-19

(44) [PsyDraw: A Multi-Agent Multimodal System for Mental Health Screening in Left-Behind Children](http://arxiv.org/abs/2412.14769v1), 2024-12-19

(45) [CodeRepoQA: A Large-scale Benchmark for Software Engineering Question Answering](http://arxiv.org/abs/2412.14764v1), 2024-12-19

(46) [Query pipeline optimization for cancer patient question answering systems](http://arxiv.org/abs/2412.14751v1), 2024-12-19

(47) [On Verbalized Confidence Scores for LLMs](http://arxiv.org/abs/2412.14737v1), 2024-12-19

(48) [Creation of AI-driven Smart Spaces for Enhanced Indoor Environments -- A Survey](http://arxiv.org/abs/2412.14708v1), 2024-12-19

(49) [LLMs as mediators: Can they diagnose conflicts accurately?](http://arxiv.org/abs/2412.14675v1), 2024-12-19

(50) [Analysis and Visualization of Linguistic Structures in Large Language Models: Neural Representations of Verb-Particle Constructions in BERT](http://arxiv.org/abs/2412.14670v1), 2024-12-19

(51) [Unveiling Uncertainty: A Deep Dive into Calibration and Performance of Multimodal Large Language Models](http://arxiv.org/abs/2412.14660v1), 2024-12-19

(52) [Length Controlled Generation for Black-box LLMs](http://arxiv.org/abs/2412.14656v1), 2024-12-19

(53) [TOMG-Bench: Evaluating LLMs on Text-based Open Molecule Generation](http://arxiv.org/abs/2412.14642v1), 2024-12-19

(54) [Learning to Generate Research Idea with Dynamic Control](http://arxiv.org/abs/2412.14626v1), 2024-12-19

(55) [How good is GPT at writing political speeches for the White House?](http://arxiv.org/abs/2412.14617v1), 2024-12-19

(56) [Is This You, LLM? Recognizing AI-written Programs with Multilingual Code Stylometry](http://arxiv.org/abs/2412.14611v1), 2024-12-19

(57) [MixLLM: LLM Quantization with Global Mixed-precision between Output-features and Highly-efficient System Design](http://arxiv.org/abs/2412.14590v1), 2024-12-19

(58) [Beyond Guilt: Legal Judgment Prediction with Trichotomous Reasoning](http://arxiv.org/abs/2412.14588v1), 2024-12-19

(59) [HiCM$^2$: Hierarchical Compact Memory Modeling for Dense Video Captioning](http://arxiv.org/abs/2412.14585v1), 2024-12-19

(60) [Simulation-Free Hierarchical Latent Policy Planning for Proactive Dialogues](http://arxiv.org/abs/2412.14584v1), 2024-12-19

(61) [CORD: Balancing COnsistency and Rank Distillation for Robust Retrieval-Augmented Generation](http://arxiv.org/abs/2412.14581v1), 2024-12-19

(62) [Sliding Windows Are Not the End: Exploring Full Ranking with Long-Context Large Language Models](http://arxiv.org/abs/2412.14574v1), 2024-12-19

(63) [CitaLaw: Enhancing LLM with Citations in Legal Domain](http://arxiv.org/abs/2412.14556v1), 2024-12-19

(64) [The Current Challenges of Software Engineering in the Era of Large Language Models](http://arxiv.org/abs/2412.14554v1), 2024-12-19

(65) [Multi-Level Optimal Transport for Universal Cross-Tokenizer Knowledge Distillation on Language Models](http://arxiv.org/abs/2412.14528v1), 2024-12-19

(66) [Cal-DPO: Calibrated Direct Preference Optimization for Language Model Alignment](http://arxiv.org/abs/2412.14516v1), 2024-12-19

(67) [PA-RAG: RAG Alignment via Multi-Perspective Preference Optimization](http://arxiv.org/abs/2412.14510v1), 2024-12-19

(68) [Do Large Language Models Defend Inferentialist Semantics?: On the Logical Expressivism and Anti-Representationalism of LLMs](http://arxiv.org/abs/2412.14501v1), 2024-12-19

(69) [FaultExplainer: Leveraging Large Language Models for Interpretable Fault Detection and Diagnosis](http://arxiv.org/abs/2412.14492v1), 2024-12-19

(70) [Moving Beyond LDA: A Comparison of Unsupervised Topic Modelling Techniques for Qualitative Data Analysis of Online Communities](http://arxiv.org/abs/2412.14486v1), 2024-12-19

(71) [DirectorLLM for Human-Centric Video Generation](http://arxiv.org/abs/2412.14484v1), 2024-12-19

(72) [Frenzy: A Memory-Aware Serverless LLM Training System for Heterogeneous GPU Clusters](http://arxiv.org/abs/2412.14479v1), 2024-12-19

(73) [Why We Build Local Large Language Models: An Observational Analysis from 35 Japanese and Multilingual LLMs](http://arxiv.org/abs/2412.14471v1), 2024-12-19

(74) [Agent-SafetyBench: Evaluating the Safety of LLM Agents](http://arxiv.org/abs/2412.14470v1), 2024-12-19

(75) [From Human Annotation to LLMs: SILICON Annotation Workflow for Management Research](http://arxiv.org/abs/2412.14461v1), 2024-12-19

(76) [Are Longer Prompts Always Better? Prompt Selection in Large Language Models for Recommendation Systems](http://arxiv.org/abs/2412.14454v1), 2024-12-19

(77) [ORBIT: Cost-Effective Dataset Curation for Large Language Model Domain Adaptation with an Astronomy Case Study](http://arxiv.org/abs/2412.14436v1), 2024-12-19

(78) [All-in-One Tuning and Structural Pruning for Domain-Specific LLMs](http://arxiv.org/abs/2412.14426v1), 2024-12-19

(79) [ChainRank-DPO: Chain Rank Direct Preference Optimization for LLM Rankers](http://arxiv.org/abs/2412.14405v1), 2024-12-18

(80) [LLMSA: A Compositional Neuro-Symbolic Approach to Compilation-free and Customizable Static Analysis](http://arxiv.org/abs/2412.14399v1), 2024-12-18

(81) [Clinical Trials Ontology Engineering with Large Language Models](http://arxiv.org/abs/2412.14387v1), 2024-12-18

(82) [ECG-Byte: A Tokenizer for End-to-End Generative Electrocardiogram Language Modeling](http://arxiv.org/abs/2412.14373v1), 2024-12-18

(83) [Memorization Over Reasoning? Exposing and Mitigating Verbatim Memorization in Large Language Models' Character Understanding Evaluation](http://arxiv.org/abs/2412.14368v1), 2024-12-18

(84) [ResQ: Mixed-Precision Quantization of Large Language Models with Low-Rank Residuals](http://arxiv.org/abs/2412.14363v1), 2024-12-18

(85) [A Survey on LLM Inference-Time Self-Improvement](http://arxiv.org/abs/2412.14352v1), 2024-12-18

(86) [Reinforcement Learning from Automatic Feedback for High-Quality Unit Test Generation](http://arxiv.org/abs/2412.14308v1), 2024-12-18

(87) [Multi-OphthaLingua: A Multilingual Benchmark for Assessing and Debiasing LLM Ophthalmological QA in LMICs](http://arxiv.org/abs/2412.14304v1), 2024-12-18

(88) [Fake News Detection: Comparative Evaluation of BERT-like Models and Large Language Models with Generative AI-Annotated Data](http://arxiv.org/abs/2412.14276v1), 2024-12-18

(89) [Thinking in Space: How Multimodal Large Language Models See, Remember, and Recall Spaces](http://arxiv.org/abs/2412.14171v1), 2024-12-18

(90) [MetaMorph: Multimodal Understanding and Generation via Instruction Tuning](http://arxiv.org/abs/2412.14164v1), 2024-12-18

(91) [Syzygy: Dual Code-Test C to (safe) Rust Translation using LLMs and Dynamic Analysis](http://arxiv.org/abs/2412.14234v1), 2024-12-18

(92) [TheAgentCompany: Benchmarking LLM Agents on Consequential Real World Tasks](http://arxiv.org/abs/2412.14161v1), 2024-12-18

(93) [Advanced Reasoning and Transformation Engine for Multi-Step Insight Synthesis in Data Analytics with Large Language Models](http://arxiv.org/abs/2412.14146v1), 2024-12-18

(94) [LLMs can realize combinatorial creativity: generating creative ideas via LLMs for scientific research](http://arxiv.org/abs/2412.14141v1), 2024-12-18

(95) [GLIDER: Grading LLM Interactions and Decisions using Explainable Ranking](http://arxiv.org/abs/2412.14140v1), 2024-12-18

(96) [Design choices made by LLM-based test generators prevent them from finding bugs](http://arxiv.org/abs/2412.14137v1), 2024-12-18

(97) [Scaling of Search and Learning: A Roadmap to Reproduce o1 from Reinforcement Learning Perspective](http://arxiv.org/abs/2412.14135v1), 2024-12-18

(98) [Alignment faking in large language models](http://arxiv.org/abs/2412.14093v1), 2024-12-18

(99) [Future Research Avenues for Artificial Intelligence in Digital Gaming: An Exploratory Report](http://arxiv.org/abs/2412.14085v1), 2024-12-18

(100) [Rango: Adaptive Retrieval-Augmented Proving for Automated Software Verification](http://arxiv.org/abs/2412.14063v1), 2024-12-18
